### PR TITLE
Redux Project - Bookstore, Milestone 5

### PR DESCRIPTION
**Pull request by tanzila-abedin**
I have done the following for this milestone :

- Implement the styling of the different components based on the design on ZEPLIN.

Thank you in advance for the review. ⭐

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,306 @@
...
+  line-height: 24px;
+  text-align: left;
+  border-right: 1px solid #e5e7eb;
+}
+
+/* .book-progress {
+  display: flex;
+  border-right: 0.1px solid rgb(240, 240, 240);
+  padding-right: 1rem;
+} */
```
Comment (rukundoeric) [inline src/index.css]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,306 @@
...
+
+.category {
+  color: #121212;
+  font-family: 'Montserrat Bold', sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: left;
+  text-transform: capitalize;
+}
```
Comment (rukundoeric) [inline src/index.css]: - [x] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_11_4_2021_2-08-53 AM |
   -  The app should not be full-screen. It must be like a card in the center of the screen just like it is on the mockup design. Kindly fix it.
   - The selector should be in the toolbar, that's what that category text in the toolbar is there for. it is there to represent the category selector.
   - Toolbar's menu and icons, books list items and the create book form must be horizontally aligned, just like there are in mockup design.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @tanzila-abedin, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/index.css [hunk 3]
```diff
@@ -1,27 +1,300 @@
+
```
Comment (eliasecasta) [inline src/index.css:1]: - [x] As the last reviewer mentioned, please align the navbar elements to the rest of the ones below them to comply with the project requirements and to make it look similar to the design provided.
- [x] Please also position your category filter droplist in the position of the "categories" text, replace the text with your filter element.

image

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hi, @tanzila-abedin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 3 above)
Comment (tanzila-abedin) [reply src/index.css:1]: Hi @e71az 

I have made those changes already you are requesting, kindly have a look.
bookstore

Comment (eliasecasta) [reply src/index.css:1]: Hello Tanzila, I didn't see them reflected in my last review, sorry.

Now I can actually see the categories change that you did which is great, however, there is an issue with your navbar, depending on the zoom used the contents inside it get separated or together, I will attach some screenshots to better explain what I mean:

100% zoom:

image

125% zoom:

image

150% zoom:

image

175% zoom:

image

In order to see the navbar as we are expecting it, I have to go as far as 175% zoom, notice that the navbar elements are the only ones changing position, the rest of your page is static which is great for this milestone

- [x] Please fix this small detail to get the project approved, good luck 👍

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hi, @tanzila-abedin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oloomoses — APPROVED** [review]
Hi @tanzila-abedin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**